src/app/shared/components/add-external-cluster-dialog/steps/credentials/provider/gke/template.html

```diff
@@ -0,0 +1,26 @@
+<!--
+Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<form [formGroup]="form"
+      fxLayout="column">
+  <mat-form-field fxFlex>
+    <mat-label>Service Account</mat-label>
+    <input matInput
+           [formControlName]="Controls.ServiceAccount"
+           type="text"
```

Comment:
Shouldn't we use `type="password"` to "hide" the credentials?

Reply:
Isn't it base64 encoded?

Reply:
true, good point. 
just wondering if we want to handle it the same as in the wizard, where we never display any "real" characters for these kind of fields.

Reply:
It is.

Reply:
Changed, as we use "password" in wizard too. @kgroschoff good spot :)